src/services/render/index.tsx

```diff
@@ -146,20 +146,33 @@ export const renderChildren = (
   stylesheets: StyleSheets,
   onUpdate: HvComponentOnUpdate,
   options: HvComponentOptions,
+) => {
+  return renderChildNodes(
```

Comment:
Should we still check `element.childNodes` here? not sure if that's needed but it was in the original code.

Reply:
https://github.com/Instawork/hyperview/pull/985/commits/c673002032503500bdea17cfcff86b4b5d47551a better safe than sorry!